src/messages/layout.js

```diff
@@ -0,0 +1,10 @@
+import { gettext, singleLineString } from 'utils';
+
+
+export const DUPLICATE_XPI_ENTRY = {
+  code: 'DUPLICATE_XPI_ENTRY',
+  message: gettext('Package contains duplicate entries'),
```

Comment:
It's an easy find/replace job later, but we can probably use `_` for this since we aren't likely to be using Underscore.js :wink: